content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,213 @@
+module.exports = {
+  // ... other configuration
+  externals: [nodeExternals()],
+  // ... other configuration
+}
 ```
 
-### Result
-
-If the configuration is successful, you see the following result in the terminal.
-
-```terminal
-✔ Profile "default" has been configured
+Update the `plugins` property in your `webpack.config.js` file to include the `copy-webpack-plugin`. This plugin will allow you to copy your `schema.prisma` file into your bundled code.
+
+```javascript file=webpack.config.js highlight=2,7-13;normal;
+const nodeExternals = require('webpack-node-externals')
+const CopyPlugin = require('copy-webpack-plugin')
+
+module.exports = {
+  // ... other configuration
+  externals: [nodeExternals()],
+  plugins: [
+    new CopyPlugin({
+      patterns: [
+        { from: './node_modules/.prisma/client/schema.prisma', to: './' }, // you may need to change `to` here.
+      ],
+    }),
+  ],
+  // ... other configuration
+}
 ```
 
-## 5. Deploy your project
-
-You have finished the configuration, so you can now deploy your project with `serverless deploy`.
+<Admonition>
 
-### Steps
+Depending on how your application is bundled, you may need to copy the schema file to a location other than `./`. Use the `serverless package` command to package your code locally so you can review where your schema should be put.
 
-Use the following command:
-
-```terminal
-serverless deploy
-```
+</Admonition>
 
-### Result
-
-Serverless creates the AWS resources and uploads your code, then outputs the service information. For example:
-
-```code no-copy
-Service Information
-service: prisma-aws-lambda-example
-stage: dev
-region: us-east-1
-stack: prisma-aws-lambda-example-dev
-resources: 39
-api keys:
-  None
-endpoints:
-  GET - https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/
-  GET - https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/seed
-  GET - https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/users
-  POST - https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/users
-  GET - https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/posts
-functions:
-  status: prisma-aws-lambda-example-dev-status
-  seed: prisma-aws-lambda-example-dev-seed
-  getUsers: prisma-aws-lambda-example-dev-getUsers
-  createUser: prisma-aws-lambda-example-dev-createUser
-  getPosts: prisma-aws-lambda-example-dev-getPosts
-layers:
-  None
-```
+Refer to the [Serverless Webpack documentation](https://www.serverless.com/plugins/serverless-webpack) for additional configuration.
 
-### What's next
+### 3. Update <inlinecode>serverless.yml</inlinecode>
 
-You can now test the deployment with a call to the status endpoint. Use the following command and replace `UNIQUE_IDENTIFIER` with the corresponding part of the URL from the output you got from `serverless deploy`.
+In `serverless.yml` make sure that the `custom > webpack` block has `prisma generate` under `packagerOptions > scripts` as follows:
```

Comment:
Same as the previous comment, a short explanation on why would be helpful, given that a user's local project will already have Prisma Client generated locally